packages/dialtone-vue2/recipes/conversation_view/feed_item_row/feed_item_row.test.js

```diff
@@ -126,7 +126,7 @@ describe('DtFeedItemRow tests', () => {
         propsData = {
           ...basePropsData,
           showHeader: true,
-          avatarImageUrl: 'https://i1.sndcdn.com/avatars-000181324408-652e57-t500x500.jpg',
+          avatarImageUrl: '/avatar1.png',
```

Comment:
Is this supposed to be an actual test image?

Reply:
It does nothing on that test actually, just to avoid a warning maybe, but that file actually exists, the previous link wasn't working correctly tho.

Reply:
ah yeah, I should have commented the .story rather than the test. I just mean the image shows up like this:

![Screenshot 2025-02-03 at 10 47 21 AM](https://github.com/user-attachments/assets/3a79e90d-6f37-4605-aead-d5c396d747e0)

we should put in some sort of actual placeholder image.

Reply:
Ah right, forgot about the base, will update to append the base URL